panel/models/web_component.ts

```diff
@@ -62,11 +64,32 @@ export class WebComponentView extends HTMLBoxView {
     //     }
     // }
     eventHandler(ev: Event): void {
+        console.log("eventHandler")
+        console.log(ev);
         var event = ev.type;
         this.eventsCount[event] += 1;
         var eventsCountLastChanged: any = {};
         eventsCountLastChanged[event] = this.eventsCount[event]
         this.model.eventsCountLastChange = eventsCountLastChanged;
+
+        this.checkIfPropertiesChanged()
+    }
+
+    checkIfPropertiesChanged(): void {
+        console.log("checkIfPropertiesChanged")
+        var propertiesChange: any = {};
+        for (let property in this.model.propertiesToWatch) {
+            var oldValue: any = this.propertyValues[property];

```

Comment:
If I may offer a piece of advice, don't use `var` unless you really know how it works. `var` has really weird semantics, especially if you're accustomed to lexical scoping (who isn't), and you may cause your self a lot of unnecessary debugging. Use `let` or `const` (preferably for immutable variables), they have normal semantics as one would expect. Let the compiler transpile this to code understood by all supported browsers, though these days `let` and `const` keywords are widely supported.

Reply:
Thanks. I will change. I don't use TypeScript very often. So I hoped someone like you would step in and guide me. 

Thanks. :-)

Reply:
I've now changed var to let